resources/views/partials/advanced-search/search-inputs.blade.php

```diff
@@ -1,10 +1,28 @@
-<span>
+{{-- TODO check for every TAG --}}
```

Comment:
And also fix that the select2 dropdowns don't use the entire screen width when opend